src/legacy/core_plugins/vis_default_editor/public/components/controls/time_interval.tsx

```diff
@@ -25,6 +25,81 @@ import { FormattedMessage } from '@kbn/i18n/react';
 
 import { search, AggParamOption } from '../../../../../../plugins/data/public';
 import { AggParamEditorProps } from '../agg_param_props';
+const { parseEsInterval, InvalidEsCalendarIntervalError } = search.aggs;
+
+function getInvalidEsMessage(interval: string) {
+  return i18n.translate(
+    'visDefaultEditor.controls.timeInterval.invalidCalendarIntervalErrorMessage',
+    {
+      defaultMessage: 'Invalid calendar interval: {interval}, value must be 1',
```

Comment:
`value must be 1`
I'm not sure about such a hint, I think a user will hardly understand what does this mean (at least new users will be puzzled).
I would use here the sentence from the doc:
`Calendar intervals can only be specified in "singular" quantities of the unit (1d, 1w, 1M, etc).`

But let's better ask @gchaps

Reply:
I just used the same as the original message of `InvalidEsCalendarIntervalError` https://github.com/elastic/kibana/blob/master/src/plugins/data/common/search/aggs/date_interval_utils/invalid_es_calendar_interval_error.ts#L31

Reply:
In that case you could use the `error.message` of `InvalidEsCalendarIntervalError`, why do you need to duplicate mesages?

Reply:
I updated `isValidCalendarInterval`,  please take a look.